org.pvermeer.WebAppHub.yml

```diff
@@ -0,0 +1,39 @@
+id: org.pvermeer.WebAppHub
+runtime: org.gnome.Platform
+runtime-version: "49"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: web-app-hub
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=xdg-data/applications:create
+  - --filesystem=xdg-data/flatpak/app:ro
+  - --filesystem=/var/lib/flatpak/app:ro
+  - --filesystem=~/.var/app:create
```

Comment:
why are these needed?

Reply:
Ok, was not sure this was needed for new submissions. I've created a PR in flathub-infra/flatpak-builder-lint#811.